src/reducers/socket.js

```diff
@@ -0,0 +1,33 @@
+import {
+  SET_SOCKET_STATUS,
+  SET_SOCKET_ERROR,
+  RESET_SOCKET_STATE,
+} from '../constants/ActionTypes'
+
+const defaultState = {
+  status: null,
```

Comment:
I don't think we're using this right now, are there plans to in the future?

Reply:
Yeah I mentioned this in the original comment for this PR.